src/internal-context/load.ts

```diff
@@ -158,7 +156,7 @@ const createInternalContext = async (moduleOptions: ModuleOptions, nuxt = useNux
 
     const template = !meta.disableHMR || !ctx?.meta?.disableHMR
       ? addTemplate({
-        filename: 'tailwind.config.cjs',
+        filename: 'tailwind.config.mjs',
```

Comment:
main issue that kept coming that stopped me from making this change are 500 status errors from vite server, and I realised this is due to exposeConfig. so in fact I need to distinguish those templates further.